code/procs/accents.dm

```diff
@@ -2045,3 +2045,322 @@ var/list/zalgo_mid = list(
+				new_string = "Rh"	// "Rhench" rhies
+				used = 2
+
+		if("h")
+			if(R.prev_char == "")	// First "h" in a word
+				new_string = "rh"	// rhelp mre!
+				used = 2
+		if("H")
+			if(R.prev_char == "")	// First "H" in a word
+				new_string = "Rh"	// Rhr
+				used = 2
+
+
+		if("i")
+			if(R.prev_char == "" && R.next_char == "")			// i
+				new_string = "rhi"
+				used = 3
+			else if(R.prev_char == "" && R.next_char == "'")	// i'm
+				new_string = "rhi"
+				used = 3
+			else if(R.prev_char == "" && R.next_char != "")		// First "i" in a word, and it ain't "i" or "i'm"
+				new_string = "rhy"
+				used = 3
+		if("I")
+			if(R.prev_char == "" && R.next_char == "")			// I
+				new_string = "Rhi"
+				used = 3
+			else if(R.prev_char == "" && R.next_char == "'")	// I'm
+				new_string = "Rhi"
+				used = 3
+			else if(R.prev_char == "" && R.next_char != "")		// First "i" in a word, and it ain't "I" or "I'm"
+				new_string = "Rhy"
+				used = 3
+
+		if("j")
+			new_string = "rrhr"
+			used = 4
+		if("J")
+			new_string = "Rrhr"
+			used = 4
+
+		if("k")
+			if(R.prev_char == "")	// First "k" in a word
+				new_string = "r"	// krakken = rakken
+				used = 1
+		if("K")
+			if(R.prev_char == "")	// First "K" in a word
+				new_string = "R"	// Krakken = Rakken
+				used = 1
+
+		if("l")
+			if(R.prev_char == "" && R.next_char == "")			// i
+				new_string = "rhi"
+				used = 3
+			else if(R.prev_char == "" && R.next_char == "'")	// i'm
+				new_string = "rhi"
+				used = 3
+			else if(R.prev_char == "" && R.next_char != "")		// First "i" in a word, and it ain't "i" or "i'm"
+				new_string = "rhy"
+				used = 3
+		if("l")
+			if(lowertext(R.next_char) == "l")		// "ll" - hello = herghlo
+				new_string = "rghl"
+				used = 4
+			else if(R.prev_char == "")				// first "l" of a word - lightly = wrightly
+				new_string = "wr"
+				used = 2
+			else
+				new_string = "rrl"					// any other "l" - heal = hearrl
+				used = 3
+		if("L")
+			if(lowertext(R.next_char) == "l")		// "ll" - hello = herghlo
+				new_string = "Rghl"
+				used = 4
+			else if(R.prev_char == "")				// first "l" of a word - lightly = wrightly
+				new_string = "Wr"
+				used = 2
+			else
+				new_string = "Rrl"					// any other "l" - heal = hearrl
+				used = 3
+
+		if("m")
+			if(R.prev_char == "")		// First "m" of a word
+				new_string = "mr"		// mime = mrime
+				used = 2
+		if("M")
+			if(R.prev_char == "")		// First "M" of a word
+				new_string = "Mr"		// Mime = Mrime
+				used = 2
+
+		if("n")
+			if(R.prev_char == "")		// First "n" of a word
+				new_string = "rh"		// nine = rhine
+				used = 2
+		if("N")
+			if(R.prev_char == "")		// First "N" of a word
+				new_string = "Rh"		// Nine = Rhine
+				used = 2
+
+		if("o")
+			if(lowertext(R.next_char) == "o")	// "oo"
+				new_string = "rooh"				// oops = roohps
+				used = 4
+			if(lowertext(R.next_char) == "h")	// "oh"
+				new_string = "roh"				// oh = roh
+				used = 3			
+			else								// all other "o"
```

Comment:
wouldn't it apply only to the if case on line 2206, making condition on the line 2203 unused (since either "roh", or "rho" are fired)?